libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFApplication.h

```diff
@@ -23,13 +23,15 @@
  */
 
 #import <UIKit/UIKit.h>
+#import <SalesforceSDKCore/SalesforceSDKConstants.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Mobile SDK extension of the UIApplication object.  Currently just gathers
  * date/time information about user events.
  */
+SFSDK_DEPRECATED(11.0, 12.0, "Will be removed")
 @interface SFApplication : UIApplication
```

Comment:
Now the screen lock is based on time since backgrounding instead of activity, I think both of these classes can be removed. Bonus is that SDK apps could use `@main` instead of initializing `UIApplicationMain` with `SFApplication`